WebContent/index.html

```diff
@@ -16,6 +16,33 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=0, maximum-scale=1, minimum-scale=1">
     <title>JES Explorer</title>
 
+    <script>
+      if(exports){
+        var ZoweZLUX_tempExports = exports;
+      }
+      var exports = {"__esModule": true};
+    </script>
+    <script type="text/javascript" src="../../../../../lib/org.zowe.zlux.logger/0.9.0/logger.js"></script>
+    <script type="text/javascript" src="../../../plugins/org.zowe.zlux.bootstrap/web/iframe-adapter.js"></script>
+    <script>
+      function handleMessageAdded(message){
```

Comment:
Are the functions stubs necessary? if they're not being used lets not checkin unreachable code...

Reply:
functions stubs are required, after we send message notification bar call us back, if stub not there it produce a crash in console.